README.md

```diff
@@ -1,51 +1,46 @@
-  * Zunächst besucht man die Seite <https://miktex.org/download>
-  * Hier lädt man sich die Version für sein Betriebssystem herunter. Hierbei lieber den installer als die portable edition nutzen.
-  * Beim Installieren sollten folgende Einstellungen beachtet werden: 
-    * Preffered paper: <span style="color:red">A4</span>
-    * Install missing packages on-the-fly: <span style="color:red">Yes</span> 
-      * Diese Option führt dazu, dass man nicht jedes mal bestätigen muss wenn bestimmte Packages fehlen. Später beim ausführen merkt man wie viele eigentlich installiert werden müssen für dieses Template.
-  * Danach kann man MikTeX ganz einfach installieren.
-  * Anschließend sollte man einmal MikTeX öffnen um nach updates zu suchen (Der erste Start gibt dies auch meist als Fehlermeldung aus.)
-  * In dem Reiter "Updates" kann man dann einfach alle verfügbaren updates installieren. So hat man immer sicher die neuste Version.
-
-## Vorbereitung
-
-* Zunächst braucht man eine "reports.json"-file 
-  * In zukünftigen Updates kann man den Speicherort dynamisch angeben. Aktuell muss diese im main directory liegen
-  * In dem repository liegt eine Beispiel Datei um den Aufba zu erklären. Bei weiteren Fragen den Source code lesen oder an KP wenden.
-* Überprüfen ob alle Tools aus den *Requirements* installiert sind.
-* Danach die *setup.py* ausführen und einige Zeit warten.
-* Die Ausgabe befindet sich in dem Ordner "report_output\_\*"
-* Daten müssen in config.py bzw. reports/report.tex angepasst werden.
-
-## Informationen & Tips
-
-* Wenn Zeichen wie z.B. '&' nicht richtig angezeigt werden, liegt das Problem meist an LaTeX selber. Ändert man '&' zu '\\\\&' wird dieses Problem behoben. 
-  * Dies gilt auch z.B. für ["]
-  * Da json hier als Format genutzt wird muss vor anführungszeichen ein Backslash [\\].
-  * ["] -> [\\"]
-* Einträge die mit "$s " markiert werden gelten als Berufsschuleinträge und werden deshalb anders im Bericht angezeigt
-* Der Marker "::" trennt einen Eintrag und macht den ersten Teil fettgedruckt und ersetzt "::" mit " - "
-* Mit Hilfe von folgender Notation kann man farben nutzen 
-  * \<color:red\>Text\<color:end\>
-* In config.py befinden sich shortcuts die man nutzen kann um spezeille Events zu definieren
\ No newline at end of file
+This project does need [Python](https://www.python.org/) and [LaTeX](https://www.latex-project.org/) to work corectly.
+
+<a href="https://www.python.org/"><img src="https://skillicons.dev/icons?i=python" alt="Python"></a>
+<a href="https://www.latex-project.org/"><img src="https://skillicons.dev/icons?i=latex" alt="LaTeX"></a>
+
+### Python
+Python can be installed through the provided installer or package on theri [download page](https://www.python.org/downloads/).
+
+### LaTeX
+
+If you have installed LaTeX please make sure the terminal command `pdflatex` is working correctly since this is the building command used for the reports.
+
```

Comment:
check for any updates if having problems building

Reply:
I rewrote it like this:
- "If you encounter problems building with your LaTeX installation please make sure to check for any updates and that `pdflatex` is in your path and accesible since this is the building command used for the reports."

Thought about it and it probably makes sense to mention the path instead of the command here.